src/osdp_pd.c

```diff
@@ -983,7 +987,18 @@ static void osdp_pd_update(struct osdp_pd *pd)
 	case OSDP_PD_STATE_PROCESS_CMD:
 		ret = pd_receve_packet(pd);
 		if (ret == OSDP_PD_ERR_NO_DATA &&
-		    osdp_millis_since(pd->tstamp) < OSDP_RESP_TOUT_MS) {
+			osdp_millis_since(pd->tstamp) < OSDP_RESP_TOUT_MS) {
+			break;
+		}
+		if (ret == OSDP_PD_ERR_IGNORE) {
+			/* Ignore this message */
+			pd->state = OSDP_PD_ERR_NONE;
```

Comment:
These two lines are also not needed; `pd->state` is overwritten in the next IF-ESLE block anyway.